src/backend/backend.ts

```diff
@@ -68,12 +68,16 @@ const messageBuffer = new MessageQueue<Message<any>>();
 /// NOTE(cbond): We collect roots from all applications (mulit-app support)
 let previousTree: MutableTree,
   previousCount: number,
+  initialModules: { [key: string]: any },
   onMouseOver,
   onMouseDown;
 
-const parseInitialModules = (roots: Array<any>) => {
-  if (roots.length) {
-    messageBuffer.enqueue(MessageFactory.ngModules(parseModules(roots[0])));
+const parseInitialModules = () => {
+  const roots = getAllAngularRootElements().map(r => ng.probe(r));
+  if (!initialModules && roots.length) {
```

Comment:
Discussed with Steven, he said we should never have to worry about a.length being a string or initialModules being a string, so code as it's written would work correctly.